grpc-mqtt.cabal

```diff
@@ -147,7 +147,7 @@ test-suite test
   ghc-options:
     -threaded
     -rtsopts
-    "-with-rtsopts=-N4 -I2 -qb0 -G1"
+    "-with-rtsopts=-N4"
```

Comment:
No, the tests fail on my VM with those settings.

Reply:
Looks like these flags were added recently. I think this is fine to remove them since it's just the test-suite, but I don't know the original purpose. Maybe @riz0id can chime in here.

It is odd though that the tests fail for you with these settings...

Reply:
Just ran the tests myself and I also get test failures when using these extra RTS options.

Reply:
@rkaippully those RTS flags are incorrect and were left as a result of me being sloppy while I was making some quick changes. I think just toggling `-N4` is correct.